### DISCUSSION TITLE
Why is goreleaser connecting to goreleaser.com?

### DISCUSSION THREAD

**Discussion by david-mkl** [goreleaser/goreleaser#3260] [Q&A] [answered]
I noticed that `goreleaser` reaches out to `goreleaser.com` at some point during the `release` command. I assume it does this on other commands too, but I've only tried `release`. (For context, I'm releasing to github and homebrew if that makes a difference.)

I've been searching around the code looking for when this happens, but I can't seem to find it. I'm mostly curious why this is necessary.

Update:
I'm also using the goreleaser github action to run goreleaser, and found this https://github.com/goreleaser/goreleaser-action/blob/master/src/github.ts#L41.  I wonder if this is the source of the network call and not goreleaser itself?

**Comment by caarlos0** [comment, marked as answer]
goreleaser pro does a `GET https://goreleaser.com/static/releases-pro.json` to see if it is the latest version...

the action also does a `GET` to either `https://goreleaser.com/static/releases-pro.json` or `https://goreleaser.com/static/releases.json` to get the latest goreleaser version...

unless I'm forgetting something, that is it.

I hope that clarifies things :)

FWIW: goreleaser.com has no tracking whatsoever :)
